src/runner.py

```diff
@@ -759,6 +828,22 @@ def _apt_install(self, packages: Iterable[str]) -> None:
 
         self.instance.execute(["/usr/bin/apt-get", "clean"])
 
+    def _snap_install(self, snaps: Iterable[Snap]) -> None:
+        """Installs the given snap packages.
+
+        This is a temporary solution to provide tools not offered by the base ubuntu image. Custom
+        images based on the GitHub action runner image will be used in the future.
+
+        Args:
+            snaps: snaps to be installed.
+        """
+        if self.instance is None:
```

Comment:
Is this actually possible?

Reply:
This is mainly to make mypy happy. I followed the way it was done in the other methods.